Packs/SailPointIdentityIQ/Integrations/SailPointIdentityIQ/SailPointIdentityIQ_description.md

```diff
@@ -0,0 +1,2 @@
+## SailPoint-IdentityIQ
+SailPoint IdentityIQ context pack enables XSOAR customers to utilize the deep, enriched contextual data in the SailPoint Predictive Identity platform to better drive identity-aware security practices.
```

Comment:
This file is usually used for troubleshooting or provide details on how to get access to the product. It's only shown when configuring the integration. I suggest adding the API details here, or to move this description to the pack readme and/or metadata description.

Reply:
Removed this from the md file. This is already in the metadata description.